### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by Pulya10c**

### CODE REVIEW

File: src/components/films/film-card/film-card.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    <p className={classes.title}>{film.title}</p>
+                    {film.genres.join(' ')}
+                </div>
+                <p className={classes.year}>{film.release_date}</p>
+            </div>
+        </div>
+    )
+}
+
+FilmCard.propTypes = {
```
Comment (kkatsuba) [inline src/components/films/film-card/film-card.tsx:54]: actually not see any reason to use them with ts

File: src/components/films/film-card/film-card.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}) => {
+    const [image, setImage] = useState<string>()
+    const [isMenuShown, showMenu] = useState<boolean>()
+    const classes = filmCardStyles()
+
+    useEffect(() => {
+        fetch(film.poster_path)
+            .then((res) => res.blob())
+            .then((imageBlob) => setImage(URL.createObjectURL(imageBlob)))
+            .catch(() => setImage(DEFAULT_IMAGE))
```
Comment (kkatsuba) [inline src/components/films/film-card/film-card.tsx:21]: not imported

File: src/components/films/films.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import { FilmsActionPanel } from './films-action-panel/films-action-panel'
+import { filmsStyles } from './films.style'
+import { FilmsSearching } from './films-searching/films-searching'
+import { ErrorBoundary } from '../../shared/components/error-boundary/error-boundary'
+
+export const Films: React.FunctionComponent<{
+    filmsService: FilmsService
+}> = ({ filmsService }) => {
+    const [films, setLoadedFilms] = useState<FilmModel[]>()
+    const isLoading = !films
```
Comment (kkatsuba) [inline src/components/films/films.tsx]: not used

**Review by kkatsuba — APPROVED** [review]

File: src/components/films/film-card/film-card.tsx (hunk 1 above)
Comment (Pulya10c) [reply src/components/films/film-card/film-card.tsx:54]: I agree, but there is a requirement in the task to use propTypes

File: src/components/films/film-card/film-card.tsx (hunk 2 above)
Comment (Pulya10c) [reply src/components/films/film-card/film-card.tsx:21]: Thanks, done

File: src/components/films/films.tsx (hunk 3 above)
Comment (Pulya10c) [reply src/components/films/films.tsx]: Thanks, done

**Pull request merged**